Applications/Calendar/Calendar.cpp

```diff
@@ -0,0 +1,83 @@
+#include "Calendar.h"
+#include <time.h>
+
+String Calendar::get_name_of_month(int month)
+{
+    switch (month) {
```

Comment:
This could be less verbose if you'd put these names into a const array, then return them like `return month_names[month - 1];` or something after some bounds checking.